src/pages/market/ui/PaginationBar.jsx

```diff
@@ -0,0 +1,47 @@
+import './PaginationBar.css';
+import { ReactComponent as LeftArrow } from '../../../images/icons/arrow_left.svg';
+import { ReactComponent as RightArrow } from '../../../images/icons/arrow_right.svg';
+
+const PaginationBar = ({ totalPageNum, activePageNum, onPageChange }) => {
+  const maxVisiblePages = 5;
```

Comment:
```suggestion
  const MAX_VISIBLE_PAGES = 5;
```
잘 해주셨습니다! 어떤 숫자인지 변수로 잘 처리해주셨어요!
이렇게 변하지 않는 값을 의미하는 네이밍 컨벤션은 UPPER_SNAKE_CASE로 보통 이름짓습니다 참고해 주세요~